src/core/Akka.TestKit/EventFilter/TestEventListener.cs

```diff
@@ -78,9 +78,9 @@ protected override bool Receive(object message)
                 case UnhandledMessage un:
                 {
                     var rcp = un.Recipient;
-                    var warning = new Warning(rcp.Path.ToString(), rcp.GetType(), "Unhandled message from " + un.Sender + ": " + un.Message);
-                    if(!ShouldFilter(warning))
-                        Print(warning);
+                    var info = new Info(rcp.Path.ToString(), rcp.GetType(), "Unhandled message from " + un.Sender + ": " + un.Message);
```

Comment:
But this is a `Warning` in the JVM. Are we sure this is the root cause of the issue?

Reply:
As i see in `Akka.Event.DeadLetterListener` message `UnhandledMessage` is handled as Info message instead of `TestEventListener`.

Reply:
https://github.com/akkadotnet/akka.net/blob/1b96294db2de1a85e670e3829c7c93a1f7a93687/src/core/Akka/Event/DeadLetterListener.cs#L192

Reply:
I found solution for #6316 and it's blocked by this PR. When it will be merged i can create MR for it.